### PR TITLE
gradle 2.7

### PR DESCRIPTION
**Pull request by seanjreilly**
Gradle 2.7 has been released

### CODE REVIEW

**Comment by dunn** [comment]
Any chance of adding a quick test while you're at it? Just something to test basic functionality would be great!

**Comment by seanjreilly** [comment]
@dunn: Sure. I've added a quick test that prints version information and checks that it prints
the appropriate version number (heavily inspired by a few existing formulae, as I haven't written a test for a homebrew recipe before). Does that look ok?

File: Library/Formula/gradle.rb [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  sha256 "cde43b90945b5304c43ee36e58aab4cc6fb3a3d5f9bd9449bb1709a68371cb06"
 
   def install
     libexec.install %w[bin lib]
     bin.install_symlink libexec+"bin/gradle"
   end
+
+  test do
+    output = shell_output("#{bin}/gradle --version")
+    assert output.include?("Gradle #{version}")
```
Comment (DomT4) [inline Library/Formula/gradle.rb]: We use `assert_match` over `assert output.include?` these days. Should be something like:

```
output = shell_output("#{bin}/gradle --version")
assert_match /Gradle #{version}/, output
```

**Comment by seanjreilly** [comment]
@DomT4: done. I have to say that this requirement wasn't obvious — I couldn't find any reference to it on http://brew.sh/, https://git.io/brew-docs, or https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md.

Maybe a documentation update would help?

**Comment by jaredsburrows** [comment]
@dunn @seanjreilly Let's get this in.

**Comment by seanjreilly** [comment]
@jaredsburrows I agree. I think the PR has been ready to go for a while and I'm a bit surprised that it's been this involved, TBH. All of the submissions I've made in the past for previous gradle versions were handled quickly (impressively quickly, even) and this one didn't seem any different to me when I first submitted it.

Is there any further reason to delay this PR that I'm not seeing?

**Comment by DomT4** [comment]
We've been having monster CI problems this week, which has resulted in this build only being passed [a few hours ago](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/33340/). The problem wasn't your fault, but appreciate patience when the CI is acting up. I'll merge it in a second.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/4e5200fcb18c52f0ca54db83e769863465aeab76. Thank you for your contribution to Homebrew @seanjreilly!

**Comment by seanjreilly** [comment]
Oh, that's what it was? No problems then. I thought there was another problem with the test I had added. Thanks for the merge!

**Comment by DomT4** [comment]
Yeah. Nothing on your side. We've just had a *fun* week with our automated testing.

**Comment by jaredsburrows** [comment]
Great. This is in.

**Pull request closed without merging**